packages/fxa-auth-server/lib/senders/emails/templates/subscriptionDowngrade/en.ftl

```diff
@@ -10,7 +10,7 @@ subscriptionDowngrade-content-switch = You have successfully switched from { $pr
 # $paymentAmountNew (String) - The amount of the new subscription payment, including currency, e.g. $10.00
 # $productPaymentCycle (String) - The interval of time from the end of one payment statement date to the next payment statement date, e.g. month
 # $paymentProrated (String) - The one time fee to reflect the higher charge for the remainder of the payment cycle, including currency, e.g. $10.00
-subscriptionDowngrade-content-charge = Starting with your next bill, your charge will change from { $paymentAmountOld } per { $productPaymentCycle } to { $paymentAmountNew }. At that time you will also be given a one-time credit of { $paymentProrated } to reflect the lower charge for the remainder of this { $productPaymentCycle }.
+subscriptionDowngrade-content-charge = Starting with your next bill, your charge will change from { $paymentAmountOld } to { $paymentAmountNew } per { $productPaymentCycle }. At that time you will also be given a one-time credit of { $paymentProrated } to reflect the lower charge for the remainder of this { $productPaymentCycle }.
```

Comment:
Darn, [my original comment in your other PR](https://github.com/mozilla/fxa/pull/11410#discussion_r773507992) applies here. We have to update the string IDs.